torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -89,33 +89,34 @@ namespace diagnostics = ::torch::onnx::diagnostics;
 
 c10::ShapeSymbol ONNXDimToShapeSymbol(
     const onnx::TensorShapeProto_Dimension& dim,
-    SymbolDimMap& symbol_dim_map) {
+    SymbolDimMap& symbol_dim_map,
+    DimSymbolMap& dim_symbol_map) {
```

Comment:
Helpful to annotate the param?

Reply:
As other prs I would include the rationale for this param as a comment

Reply:
Good thinking! I get confused myself if I haven't look at it for a while. I've added a comment that explains why we need both ways.

In writing this, I actually think an argument could be made that `symbol_dim_map` could be removed and created once at the end, since `dim_symbol_map` is the look-up direction that seems to be happening actively during the export. Anyway, it would need a deeper look, so for now I'm not going to push on this.